src/main/java/uk/gov/hmcts/reform/pip/rules/model/Hearing.java

```diff
@@ -0,0 +1,56 @@
+package uk.gov.hmcts.reform.pip.rules.model;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import lombok.Data;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+
+/**
+ * Model which represents a hearing for a court.
+ */
+@Data
+public class Hearing {
+
+    /**
+     * The ID for the hearing.
+     */
+    private Integer hearingId;
+
+    /**
+     * The ID for the court the hearing is for.
+     */
+    private Integer courtId;
+
+    /**
+     * The date of the hearing.
+     */
+    @JsonFormat(pattern = "MM/dd/yyyy")
```

Comment:
Should we use UK date format "dd/MM/yyyy"?

Reply:
Updated to use dd/MM/yyyy format